### PR TITLE
Create 2020-03-25-kubernetes-1.18-release-announcement.md

### PR DESCRIPTION
**Pull request by jiaj12**
add zh-trans 2020-03-25-kubernetes-1.18-release-announcement.md

### CODE REVIEW

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+### Release Team
+-->
+### 发布团队
+
+<!--
+This release is made possible through the efforts of hundreds of individuals who contributed both technical and non-technical content. Special thanks to the [release team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.18/release_team.md) led by Jorge Alarcon Ochoa, Site Reliability Engineer at Searchable AI. The 34 release team members coordinated many aspects of the release, from documentation to testing, validation, and feature completeness. 
+-->
+通过数百位贡献了技术和非技术内容的个人的努力，使本次发行成为可能。 特别感谢 [发布团队](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.18/release_team.md)，由 Searchable AI 的网站可靠性工程师 Jorge Alarcon Ochoa 领导。 34 位发布团队成员协调了发布的各个方面，从文档到测试，验证和功能完整性。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:181]:
```suggestion
通过数百位贡献了技术和非技术内容的个人的努力，使本次发行成为可能。 特别感谢由 Searchable AI 的网站可靠性工程师 Jorge Alarcon Ochoa 领导的[发布团队](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.18/release_team.md)。 34 位发布团队成员协调了发布的各个方面，从文档到测试、验证和功能完整性。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+As the Kubernetes community has grown, our release process represents an amazing demonstration of collaboration in open source software development. Kubernetes continues to gain new users at a rapid pace. This growth creates a positive feedback cycle where more contributors commit code creating a more vibrant ecosystem. Kubernetes has had over [40,000 individual contributors](https://k8s.devstats.cncf.io/d/24/overall-project-statistics?orgId=1) to date and an active community of more than 3,000 people.
+-->
+随着 Kubernetes 社区的发展壮大，我们的发布过程很好地展示了开源软件开发中的协作。 Kubernetes 继续快速获取新用户。 这种增长创造了一个积极的反馈周期，其中有更多的贡献者提交了代码，从而创建了更加活跃的生态系统。 迄今为止，Kubernetes 已有 [40,000 独立贡献者](https://k8s.devstats.cncf.io/d/24/overall-project-statistics?orgId=1) 和一个超过3000人的活跃社区。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:186]: feedback cycle

反馈回路

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+#### Why the LHC?
+-->
+#### 为什么是 LHC
+
+<!--
+The LHC is the world’s largest and most powerful particle accelerator.  It is the result of the collaboration of thousands of scientists from around the world, all for the advancement of science. In a similar manner, Kubernetes has been a project that has united thousands of contributors from hundreds of organizations – all to work towards the same goal of improving cloud computing in all aspects! "A Bit Quarky" as the release name is meant to remind us that unconventional ideas can bring about great change and keeping an open mind to diversity will lead help us innovate.
+-->
+LHC 是世界上最大，功能最强大的粒子加速器。 这是来自世界各地成千上万科学家合作的结果，所有这些都是为了促进科学的发展。 以类似的方式，Kubernetes 已经成为一个聚集了来自数百个组织的数千名贡献者–所有人都朝着在各个方面改善云计算的相同目标努力的项目！ 发布名称“ A  Bit Quarky” 的意思是提醒我们，非常规的想法可以带来巨大的变化，对开放性保持开放态度将有助于我们进行创新。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:206]:
```suggestion
LHC 是世界上最大，功能最强大的粒子加速器，是来自世界各地成千上万科学家合作的结果。所有这些合作都是为了促进科学的发展。以类似的方式，Kubernetes 已经成为一个聚集了来自数百个组织的数千名贡献者 – 所有人都朝着在各个方面改善云计算的相同目标努力的项目！ 发布名称 “A Bit Quarky” 的意思是提醒我们，非常规的想法可以带来巨大的变化，对开放性保持开放态度将有助于我们进行创新。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+#### About the designer
+-->
+#### 关于设计者
+
+<!--
+Maru Lango is a designer currently based in Mexico City. While her area of expertise is Product Design, she also enjoys branding, illustration and visual experiments using CSS + JS and contributing to diversity efforts within the tech and design communities. You may find her in most social media as @marulango or check her website: https://marulango.com
+-->
+Maru Lango 是目前居住在墨西哥城的设计师。 尽管她的专长是产品设计，但她还喜欢使用 CSS + JS 进行品牌，插图和视觉实验，并为技术和设计社区的多样性做出了贡献。 您可能会在大多数社交媒体上以 @marulango 的身份找到她，或查看她的网站： https://marulango.com
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:217]:
```suggestion
Maru Lango 是目前居住在墨西哥城的设计师。她的专长是产品设计，她还喜欢使用 CSS + JS 进行品牌、插图和视觉实验，为技术和设计社区的多样性做贡献。您可能会在大多数社交媒体上以 @marulango 的身份找到她，或查看她的网站： https://marulango.com
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+### User Highlights
+-->
+### 高光用户
+
+<!--
+- Ericsson is using Kubernetes and other cloud native technology to deliver a [highly demanding 5G network](https://www.cncf.io/case-study/ericsson/) that resulted in up to 90 percent CI/CD savings.
+- Zendesk is using Kubernetes to [run around 70% of its existing applications](https://www.cncf.io/case-study/zendesk/). It’s also building all new applications to also run on Kubernetes, which has brought time savings, greater flexibility, and increased velocity  to its application development.
+- LifeMiles has [reduced infrastructure spending by 50%](https://www.cncf.io/case-study/lifemiles/) because of its move to Kubernetes. It has also allowed them to double its available resource capacity.
+-->
+- 爱立信正在使用 Kubernetes 和其他云原生技术来交付[高标准的 5G 网络](https://www.cncf.io/case-study/ericsson/)，这可以节省多达 90％ 的CI / CD。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:229]:
```suggestion
- 爱立信正在使用 Kubernetes 和其他云原生技术来交付[高标准的 5G 网络](https://www.cncf.io/case-study/ericsson/)，这可以在 CI/CD 上节省多达 90％ 的支出。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+- Zendesk正在使用Kubernetes[运行其现有应用程序的约70％](https://www.cncf.io/case-study/zendesk/)。 它还正在构建所有新应用程序，使其也可以在 Kubernetes 上运行，从而节省了时间，提高了灵活性并加快了其应用程序开发的速度。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:230]:
```suggestion
- Zendesk 正在使用 Kubernetes [运行其现有应用程序的约 70％](https://www.cncf.io/case-study/zendesk/)。它还正在使所构建的所有新应用都可以在 Kubernetes 上运行，从而节省时间、提高灵活性并加快其应用程序开发的速度。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 7]
```diff
@@ -0,0 +1,305 @@
...
+- LifeMiles 已迁移到 Kubernetes，[降低了 50%  的基础设施开支](https://www.cncf.io/case-study/lifemiles/)。 它还使他们可以将其可用资源容量增加一倍。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:231]:
```suggestion
- LifeMiles 因迁移到 Kubernetes 而[降低了 50% 的基础设施开支](https://www.cncf.io/case-study/lifemiles/)。Kubernetes 还使他们可以将其可用资源容量增加一倍。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 8]
```diff
@@ -0,0 +1,305 @@
...
+### Ecosystem Updates
+-->
+### 生态系统更新
+
+<!--
+- The CNCF published the results of its [annual survey](https://www.cncf.io/blog/2020/03/04/2019-cncf-survey-results-are-here-deployments-are-growing-in-size-and-speed-as-cloud-native-adoption-becomes-mainstream/) showing that Kubernetes usage in production is skyrocketing. The survey found that 78% of respondents are using Kubernetes in production compared to 58% last year.
+- The “Introduction to Kubernetes” course hosted by the CNCF [surpassed 100,000 registrations](https://www.cncf.io/announcement/2020/01/28/cloud-native-computing-foundation-announces-introduction-to-kubernetes-course-surpasses-100000-registrations/).
+-->
+- CNCF发布了[年度调查](https://www.cncf.io/blog/2020/03/04/2019-cncf-survey-results-are-here-deployments-are-growing-in-size-and-speed-as-cloud-native-adoption-becomes-mainstream/) 的结果，表明 Kubernetes 在生产中的使用正在飞速增长。调查发现，有78％的受访者在生产中使用Kubernetes，而去年这一比例为 58％。
+- CNCF举办的“ Kubernetes入门”课程 [超过 100,000 注册者](https://www.cncf.io/announcement/2020/01/28/cloud-native-computing-foundation-announces-introduction-to-kubernetes-course-surpasses-100000-registrations/)。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:243]:
```suggestion
- CNCF 举办的 “Kubernetes入门” 课程有[超过 100,000 人注册](https://www.cncf.io/announcement/2020/01/28/cloud-native-computing-foundation-announces-introduction-to-kubernetes-course-surpasses-100000-registrations/)。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 9]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+### Project Velocity
+-->
+### 项目速度
+
+<!--
+The CNCF has continued refining DevStats, an ambitious project to visualize the myriad contributions that go into the project. [K8s DevStats](https://k8s.devstats.cncf.io/d/12/dashboards?orgId=1) illustrates the breakdown of contributions from major company contributors, as well as an impressive set of preconfigured reports on everything from individual contributors to pull request lifecycle times. 
+-->
+CNCF继续完善DevStats，这是一个雄心勃勃的项目，旨在可视化项目中做出的无数贡献。[K8s DevStats](https://k8s.devstats.cncf.io/d/12/dashboards?orgId=1)阐述了主要公司贡献者的贡献细目，以及一系列令人印象深刻的预配置报告，涉及个人的所有内容贡献者拉动请求生命周期时间。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:253]:
```suggestion
CNCF 继续完善 DevStats。这是一个雄心勃勃的项目，旨在对项目中的无数贡献数据进行可视化展示。[K8s DevStats](https://k8s.devstats.cncf.io/d/12/dashboards?orgId=1) 展示了主要公司贡献者的贡献细目，以及一系列令人印象深刻的预定义的报告，涉及从贡献者个人的各方面到 PR 生命周期的各个方面。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 10]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+This past quarter, 641 different companies and over 6,409 individuals contributed to Kubernetes. [Check out DevStats](https://k8s.devstats.cncf.io/d/11/companies-contributing-in-repository-groups?orgId=1&var-period=m&var-repogroup_name=All) to learn more about the overall velocity of the Kubernetes project and community.
+-->
+在过去的一个季度中，Kubernetes 贡献了 641 家不同的公司和超过 6,409 个人。 [查看 DevStats](https://k8s.devstats.cncf.io/d/11/companies-contributing-in-repository-groups?orgId=1&var-period=m&var-repogroup_name=All) 以了解有关整体的更多信息  Kubernetes 项目和社区的发展速度。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:258]:
```suggestion
在过去的一个季度中，641 家不同的公司和超过 6,409 个个人为 Kubernetes 作出贡献。 [查看 DevStats](https://k8s.devstats.cncf.io/d/11/companies-contributing-in-repository-groups?orgId=1&var-period=m&var-repogroup_name=All) 以了解有关 Kubernetes 项目和社区发展速度的信息。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 11]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+### Upcoming Release Webinar
+-->
+### 即将到来的发布的线上会议
+
+<!--
+Join members of the Kubernetes 1.18 release team on April 23rd, 2020 to learn about the major features in this release including kubectl debug, Topography Manager, Ingress to V1 graduation, and client-go. Register here: https://www.cncf.io/webinars/kubernetes-1-18/.
+-->
+在 2020 年 4 月 23 日前加入 Kubernetes 1.18 版本团队，以了解此版本的主要功能，包括kubectl 调试，Topography Manager，Ingress 接近 V1 版本毕业，和 go 客户端客户。 在此处注册：https://www.cncf.io/webinars/kubernetes-1-18/ 。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:278]:
```suggestion
在 2020 年 4 月 23 日，和 Kubernetes 1.18 版本团队一起了解此版本的主要功能，包括 kubectl debug、拓扑管理器、Ingress 毕业为 V1 版本以及 client-go。 在此处注册：https://www.cncf.io/webinars/kubernetes-1-18/ 。
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 12]
```diff
@@ -0,0 +1,305 @@
...
+
+<!--
+### Get Involved
+-->
+### 如何参与
+
+<!--
+The simplest way to get involved with Kubernetes is by joining one of the many [Special Interest Groups](https://github.com/kubernetes/community/blob/master/sig-list.md) (SIGs) that align with your interests. Have something you’d like to broadcast to the Kubernetes community? Share your voice at our weekly [community meeting](https://github.com/kubernetes/community/tree/master/communication), and through the channels below. Thank you for your continued feedback and support.
+-->
+参与 Kubernetes 的最简单方法是加入众多与您的兴趣相关的 [特殊兴趣小组](https://github.com/kubernetes/community/blob/master/sig-list.md) (SIGs) 之一。 您有什么想向 Kubernetes 社区发布的内容吗？ 参与我们的每周 [社区会议](https://github.com/kubernetes/community/tree/master/communication)，并通过以下渠道分享您的声音。 感谢您一直以来的反馈和支持。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:288]: special interest group 译为 特别兴趣小组好些，“特殊”有些怪异。

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 13]
```diff
@@ -0,0 +1,305 @@
...
+- Follow us on Twitter [@Kubernetesio](https://twitter.com/kubernetesio) for latest updates
+- Join the community discussion on [Discuss](https://discuss.kubernetes.io/)
+- Join the community on [Slack](http://slack.k8s.io/)
+- Post questions (or answer questions) on [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
+- Share your Kubernetes [story](https://docs.google.com/a/linuxfoundation.org/forms/d/e/1FAIpQLScuI7Ye3VQHQTwBASrgkjQDSS5TP0g3AXfFhwSM9YpHgxRKFA/viewform)
+- Read more about what’s happening with Kubernetes on the [blog](https://kubernetes.io/blog/)
+- Learn more about the [Kubernetes Release Team](https://github.com/kubernetes/sig-release/tree/master/release-team)
+-->
+- 在 Twitter [@Kubernetesio](https://twitter.com/kubernetesio) 关注我们，获取最新更新
+- 参与社区讨论 [讨论](https://discuss.kubernetes.io/)
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:300]:
```suggestion
- 在 [Discuss](https://discuss.kubernetes.io/) 上参与社区讨论 
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 14]
```diff
@@ -0,0 +1,305 @@
...
+<!--
+- Follow us on Twitter [@Kubernetesio](https://twitter.com/kubernetesio) for latest updates
+- Join the community discussion on [Discuss](https://discuss.kubernetes.io/)
+- Join the community on [Slack](http://slack.k8s.io/)
+- Post questions (or answer questions) on [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
+- Share your Kubernetes [story](https://docs.google.com/a/linuxfoundation.org/forms/d/e/1FAIpQLScuI7Ye3VQHQTwBASrgkjQDSS5TP0g3AXfFhwSM9YpHgxRKFA/viewform)
+- Read more about what’s happening with Kubernetes on the [blog](https://kubernetes.io/blog/)
+- Learn more about the [Kubernetes Release Team](https://github.com/kubernetes/sig-release/tree/master/release-team)
+-->
+- 在 Twitter [@Kubernetesio](https://twitter.com/kubernetesio) 关注我们，获取最新更新
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:299]:
```suggestion
- 在 Twitter 上关注我们 [@Kubernetesio](https://twitter.com/kubernetesio)，了解最新动态
```

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 15]
```diff
@@ -113,19 +113,19 @@ The alpha version of CSI Proxy for Windows is being released with Kubernetes 1.1
...
-- [支持树外 vSphere 云提供商](https://github.com/kubernetes/enhancements/issues/670)
+- [支持树外 vSphere 云驱动](https://github.com/kubernetes/enhancements/issues/670)
 - [对 Windows 负载支持 GMSA](https://github.com/kubernetes/enhancements/issues/689)
 - [对不可挂载的CSI卷跳过挂载](https://github.com/kubernetes/enhancements/issues/770)
 - [PVC 克隆](https://github.com/kubernetes/enhancements/issues/989)
-- [Moving kubectl package code to staging](https://github.com/kubernetes/enhancements/issues/1020)
-- [RunAsUserName for Windows](https://github.com/kubernetes/enhancements/issues/1043)
-- [AppProtocol for Services and Endpoints](https://github.com/kubernetes/enhancements/issues/1507)
-- [Extending Hugepage Feature](https://github.com/kubernetes/enhancements/issues/1539)
+- [移动 kubectl 包代码到筹备阶段](https://github.com/kubernetes/enhancements/issues/1020)
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:125]: staging 是 kubernetes 源码树中的一个目录，不要翻译。

File: content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md [hunk 16]
```diff
@@ -146,13 +146,13 @@ The alpha version of CSI Proxy for Windows is being released with Kubernetes 1.1
 - [Moving kubectl package code to staging](https://github.com/kubernetes/enhancements/issues/1020)
 - [CertificateSigningRequest API](https://github.com/kubernetes/enhancements/issues/1513)
 - [Extending Hugepage Feature](https://github.com/kubernetes/enhancements/issues/1539)
-- [client-go signature refactor to standardize options and context handling](https://github.com/kubernetes/enhancements/issues/1601)
+- [client-go 的关键重构来标准化选项和管理上下文](https://github.com/kubernetes/enhancements/issues/1601)
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md:149]:
```suggestion
- [client-go 的调用规范重构来标准化选项和管理上下文](https://github.com/kubernetes/enhancements/issues/1601)
```

signature 不是 significant

**Comment by tengqm** [comment]
pls squash your commits.
I think this one is ready.

**Comment by jiaj12** [comment]
got problem with squash, start a new pr #22437 @tengqm

**Comment by tengqm** [comment]
dup of #22437 
/close

**Pull request closed without merging**